### ISSUE TITLE
JG: Comments related to summary 

### ISSUE THREAD

**Issue by hturner** [Abhi-1U/texor-rjarticle#8]
Comments from @ajrgodfrey that could be incorporated in the summary or possibly introductory sections

> - Setting good standards for R Journal articles has a positive spillover effects for the R community as authors become aware of good practice and R Journal articles/supplementary material may become package vignettes.
> - There are various other opportunities for building on this work that should be mentioned
       - vignettes
       - legacy educational material
> - There are plenty of other journals that have a similar backlog of legacy articles. The arXiv accessibility project might be mentioned here. Other journals have converted their back catalogues or are working on it, e.g. journal of statistics education converted their back catalogue from Word to HTML. Particularly smaller journals without paid staff might learn from the R Journal's experience.

**Comment by dicook** [comment]
@ajrgodfrey where can I find information about JSE's conversion of articles from Word to HTML?

**Comment by ajrgodfrey** [comment]
I've acted as a reviewer and been provided with docx files from authors.  Final article is in HTML.



My own submission was LaTeX based but now appears in HTML. It was published in 2015; the link I have on file is now dead, but the article is available at https://doi.org/10.1080/10691898.2015.11889746 with a "first published online" date of 2017, which I presume is coincidental with the move to T&F.

"conversion" is perhaps the wrong word with respect to current content. T&F have processed the back-catalogue to some extent, including content off the old AmStat site.
I don't recall any song and dance about this change of host leading to a change in presentation, but I welcomed the move from pdf to HTML.

I have reviewed the working draft of my submission in pdf and in HTML, as well as the versions provided by the publisher. The pdf documents are among the best one could ever see in a STEM subject, but the reality is that the mathematical content is very low, and there are no images at all. The current HTML is a pleasure to engage with.
